aptos-move/framework/aptos-framework/sources/Block.move

```diff
@@ -84,6 +86,7 @@ module AptosFramework::Block {
                 round,
                 previous_block_votes,
                 proposer,
+                failed_proposer_indices,
```

Comment:
How do we ensure that other validators can validate failed_proposer_indices is correct and not maliciously altered by the proposer? Is this done in a separate PR?